src/main/java/millom/sandbox/service/MyClientService.java

```diff
@@ -0,0 +1,38 @@
+package millom.sandbox.service;
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import jakarta.ws.rs.core.MediaType;
+
+public class MyClientService {
+//  private static final String REST_URI
+//      = "https://mars.nasa.gov/rss/api/?feed=weather&feedtype=json&ver=1.0&category=msl";
+
+  private static final String REST_BASE_URI
+      = "https://mars.nasa.gov/rss/api/";
+
+  private static final String API_QUERY_PARAMS = "?feed=%s&feedtype=%s&ver=%.1f&category=%s";
+  private Client client;
+
+  public MyClientService(Client client) {
+    this.client = client;
+  }
+
+  public MyClientService() {
+    this.client = ClientBuilder.newClient();
```

Comment:
What about re-using the ctor that accepts a client as argument? Something like:
```
public MyClientService() {
    this(ClientBuilder.newClient());
}
```

Reply:
changed to
```
public MyClientService() {
    this(ClientBuilder.newClient());
  }
```